doc/releasenotes/12_0_0_release_notes.md

```diff
@@ -31,8 +31,6 @@ zone1-0000000103 sourcekeyspace 0 rdonly 192.168.0.134:15103 192.168.0.134:17103
 
 ## Deprecations
 
-The command `vtctl VExec` is deprecated and removed. All Online DDL commands should run through `vtctl OnlineDDL`.
-
```

Comment:
These are the release notes that shipped with 12.0.0. Since that is out the door we should not change them.
In 12.0.1 we can write new release notes that we are undo-ing the deprecation.

Reply:
Reverted